4MD/src/main/java/com/infoshareacademy/four_md/model/dto/Recipe.java

```diff
@@ -18,13 +21,19 @@
 public class Recipe {
 
     private int id;
-
+    @NotNull(message = "{validation.empty}")
+    @NotEmpty(message = "{validation.empty}")
     private String name;
+    @NotNull(message = "{validation.empty}")
+    @NotEmpty(message = "{validation.empty}")
+    private String method;
 
     private List<Ingredients> ingredientsList = new ArrayList<>();
-
+    @NotNull(message = "{validation.empty}")
+    @Min(value = 0, message = "{validation.number}")
     private int estimatedCookingTime;
-
+    @NotNull(message = "{validation.empty}")
+    @Min(value = 0, message = "{validation.number}")
     private int calories;
```

Comment:
Kiedy z interfejsu wprowadzi sie wartosci po kropce (co sie da) to strona zuca brzydki blad: Failed to convert property value of type java.lang.String to required type int for property estimatedCookingTime; nested exception is java.lang.NumberFormatException: For input string: "-1.5"